src/mc_tasks/ImpedanceTask.cpp

```diff
@@ -227,7 +252,9 @@ void ImpedanceTask::addToGUI(mc_rtc::gui::StateBuilder & gui)
                  mc_rtc::gui::ArrayLabel("filteredMeasuredWrench", {"cx", "cy", "cz", "fx", "fy", "fz"},
                                          [this]() { return this->filteredMeasuredWrench_.vector(); }),
                  mc_rtc::gui::NumberInput("cutoffPeriod", [this]() { return this->cutoffPeriod(); },
-                                          [this](double a) { return this->cutoffPeriod(a); }));
+                                          [this](double a) { return this->cutoffPeriod(a); }),
+                 // hold
```

Comment:
Also not a required comment

Reply:
Fixed in https://github.com/jrl-umi3218/mc_rtc/pull/143/commits/51d1eb078d2d27fc83085dd1e48545344e8face7